examples/tutorials/beginner/README.txt

```diff
@@ -0,0 +1,2 @@
+Beginner Tutorials
+------------------
```

Comment:
Could this "Beginner Tutorials" heading be shortened to be just "Beginner"? Same with "Advanced Tutorials" -> "Advanced". Right now it looks like this:

![image](https://user-images.githubusercontent.com/23487320/140603896-fec07fd0-f6e9-48de-ab30-d8cb946676a6.png)

Also, just wondering if "Basics" would be a better term than "Beginner" (similar to what @core-man used at https://github.com/GenericMappingTools/pygmt/issues/1191#issuecomment-817129541), considering that PyGMT experts still need to use these basic building blocks sometimes!